### ISSUE TITLE
Looking for a full version of `ebean.properties` file

### ISSUE THREAD

**Issue by xxcode-monkey** [ebean-orm/avaje-ebeanorm#639]
I'm looking for a full version ( with comments ) of `ebean.properties` file, where can I found it, thanks!

**Comment by rbygrave** [comment]
No, there is not one existing. This is a good documentation request.

If you are in a hurry ... you can review the code that reads the
properties:  ServerConfig.loadSettings()

https://github.com/ebean-orm/avaje-ebeanorm/blob/master/src/main/java/com/avaje/ebean/config/ServerConfig.java#L2268


There is also:

AutoTuneConfig
DataSourceConfig
DocStoreConfig

Cheers, Rob.

**Comment by rPraml** [comment]
#1325 could be a good start
